credsweeper/app_path.py

```diff
@@ -0,0 +1,4 @@
+# use separate file to avoid circular import issue
+from pathlib import Path
+""" Directory of credsweeper sources """
+APP_PATH = Path(__file__).resolve().parent
```

Comment:
What about move `APP_PATH` to `credsweeper/app.py`?
I think it seems more proper

``` python
...
from credsweeper.app import CredSweeper, APP_PATH
...
```

Reply:
Unfortunately it goes to circular dependency error.

Reply:
I see. Please check this later.